dev-infrastructure/docs/development-setup.md

```diff
@@ -344,7 +357,13 @@ instance_types:
 EOF
 ```
 
-3) Follow CS dev setup process:
+3) Get first-party application certificate bundle (Note that this will change when new FPA changes are introduced soon): </br>
+Run the following command to generate a file containing the base64 decoded first-party application certificate bundle
+```shell
+$ az keyvault secret download --file fpa_cert --id=https://service-kv-aro-hcp-dev.vault.azure.net/secrets/firstPartyMock/<certi_id> | base64 -d > ~/fpa_cert_decoded
```

Comment:
The command does not work.

The specified az keyvaut secret download command does not output to stdout so the base64 decoding through the pipe is not going to work.

Another way to specify this is in two commands. The first downloading the cert and the second base64 encoding.

We cleanup of the base64-encoded file should also be added as an instruction so there are no intermediate leftovers

Reply:
Alternatively, you could use `az keyvault show .... --query "value"...` to output the b64 encoded cert then pipe it.